Project/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.4.17f1
```

Comment:
This (and the manifest) were pretty old. Just affects Project, but it's annoying to keep excluding them.